Orange/__init__.py

```diff
@@ -1,6 +1,12 @@
 # This module is a mixture of imports and code, so we allow import anywhere
 # pylint: disable=wrong-import-position,wrong-import-order
 
+try:
+    from Orange.data import _variable
+except ImportError:
+    raise ImportError("Compiled libraries cannot be found.\n"
+                      "Try reinstalling the package with: pip install --no-binary Orange3") from None
```

Comment:
Break this line and then I'm merging. :)

Since you'll modify the commit anyway, do you think it would make sense to import other modules, too? It's probably possible that `_variable` would exist, but some other module would fail to compile.

In the future, we probably won't remember to add new modules here when we introduce them, but it's still better to have at least those that exist now.

This is just a thought. If you disagree, just break the line and we're done. :)

Reply:
I added the brake.
If the compile fails pip shows errors. I think the import problems lies somewhere else.
I would therefore argue that either all library work or none does so there is no point in adding all the imports.

Reply:
I meant breaking the line because of pylint, not to add a `\n` to the message, too. But this is OK as well.